client/src/components/Common/EditEvent/EditEvent.js

```diff
@@ -0,0 +1,147 @@
+import React, { Component } from "react";
+import Calender from "../../../Images/calendar_icon.png";
+import Clock from "../../../Images/clock_icon.png";
+import RoomIcon from "../../../Images/room_icon.png";
+import "./editEvent.css";
+import axios from "axios";
+import { Link } from "react-router-dom";
+import Popup from "./Popup";
+class EditEvent extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      userId: 3,
+      date: "2020-05-23T11:00:00.000Z",
+      roomId: 5,
+      room_name: "Moscow",
+      showPopup: false,
+      event_id: "",
+        event_id: "",
+        event_title: "",
+        event_author: "",
+        event_description: "",
+        event_note: "",
+    };
+  }
+
+  componentDidMount() {
+    let eventId = this.props.match.params.id;
+    axios
+      .get(`/api/edit-event/${eventId}`)
+      .then((res) => {
```

Comment:
since there is only one statment, you can omit the `{` here and just write `.then(res => this.setState({}))`

Also, pls see point 4 in https://github.com/GSG-K3/rooms-booker/pull/64#discussion_r419640515